### DISCUSSION TITLE
How to start a thread?

### DISCUSSION THREAD

**Discussion by Seeung** [CosmosOS/Cosmos#2332] [Q&A]
How to start a thread?

 Error when building `Need plug for: System.Void System.Threading.Thread.StartInternal`

**Comment by asterd-og** [comment]
you cant, since they still didnt implemented multi threading, tho there is a branch with multi threading that u can try (i just dont know how to use it)

**Reply by Seeung** [reply]
Are there alternatives to run multiple tasks at once?

**Reply by zarlo** [reply]
nope
